src/tests.cpp

```diff
@@ -1441,7 +1441,221 @@ TEST_CASE( "BSMHandler JSON Error Checking", "[ppm][filtering][error]" ) {
+
+    rapidjson::Document document;
+    document.Parse( jsonString );
+
+    // check that accelLong that is present is properly identified as present
+    bool accelLongPresent = false;
+    rapidjsonRedactor.searchForMemberByName( document, "accelLong", accelLongPresent );
+    CHECK( accelLongPresent == true );
+
+    // check that a nonsense member is properly identified as not present
+    bool nonsenseMemberPresent = false;
```

Comment:
We should probably pull this out as its own test case, validating that non-existent members aren't returned as false is a great case to call out.

Reply:
This test case has been split up!